test/els_completion_SUITE.erl

```diff
@@ -270,11 +266,11 @@ macros(Config) ->
 
   #{result := Completion1} =
     els_client:completion(Uri, 24, 1, TriggerKindChar, <<"?">>),
-  ?assertEqual(lists:sort(Expected), lists:sort(Completion1)),
+  [?assert(lists:member(E, Completion1)) || E <- Expected],
```

Comment:
Another spotted bug: for some reason, macros from included files were not returned in the completion list. This is now fixed.